src/main/java/io/cryostat/net/SslConfiguration.java

```diff
@@ -64,9 +64,12 @@ public class SslConfiguration {
         this.logger = logger;
 
         if (env.hasEnv(Variables.DISABLE_SSL)) {
-            strategy = new NoSslStrategy();
-            logger.info("Selected NoSSL strategy");
-            return;
+            String disableSslValue = env.getEnv(Variables.DISABLE_SSL);
+            if ("true".equals(disableSslValue) || "TRUE".equals(disableSslValue )) {
```

Comment:
This still does not agree with `entrypoint.bash`.

```bash
$ function test() {
> if [ $TEST = "true" ]; then
> echo accepted
> else
> echo "not accepted"
> fi
> }

$ TEST=true test
accepted

$ TEST=TRUE test
not accepted
```

Reply:
I tested with both ```CRYOSTAT_DISSABLE_SSL=true``` and ```CRYOSTAT_DISSABLE_SSL=TRUE``` and it worked as expected.

Reply:
Yes, the Java code will accept both of these options. The `entrypoint.bash` will not. This same variable controls both webserver SSL (Java code and the HTTPie test you are performing) and the SSL configuration for JMX and the Java built-in HTTP capabilities (entrypoint.bash, not captured with the HTTPie test you are performing). I will not accept this PR unless both of these places are updated to agree on what values they accept, since that is the true root cause of the original bug.

Reply:
Please, just make the Java code accept `true` and only `true` for disabling SSL, the same as what `entrypoint.bash` does.